libp2p/io/exceptions.py

```diff
@@ -0,0 +1,10 @@
+class MsgioException(Exception):
```

Comment:
Regarding the discussion in https://github.com/libp2p/py-libp2p/issues/261, would it better to change it to
```suggestion
from libp2p.exceptions import BaseLibp2pError


class MsgioException(BaseLibp2pError):
```

Reply:
yes -- missed that we had this! thanks